packages/blade/src/components/Typography/Text/__tests__/Text.web.test.tsx

```diff
@@ -53,38 +53,14 @@ describe('<Text />', () => {
   });
 
   it('should throw error when variant is "caption" and size "small" is passed', () => {
-    try {
-      const displayText = 'Displaying some text';
+    const displayText = 'Displaying some text';
+    expect(() =>
       renderWithTheme(
         // @ts-expect-error testing failure case when size='small' is passed with variant='caption'
         <Text type="normal" variant="caption" truncateAfterLines={3} size="small">
           {displayText}
         </Text>,
-      );
-    } catch (error: unknown) {
-      if (error instanceof Error) {
-        expect(error.message).toEqual(
-          `[Blade: Text]: size cannot be 'small' when variant is 'caption'`,
-        );
-      }
-    }
-  });
-
-  it('should throw error when variant is "caption" but weight "bold" is passed', () => {
-    try {
-      const displayText = 'Displaying some text';
-      renderWithTheme(
-        // @ts-expect-error testing failure case when weight='bold' is passed with variant='caption'
-        <Text type="normal" variant="caption" weight="bold">
-          {displayText}
-        </Text>,
-      );
-    } catch (error: unknown) {
```

Comment:
Should probably add this rule https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-conditional-expect.md

Reply:
Let's add

Reply:
Will do as a separate PR.